hermeto/core/resolver.py

```diff
@@ -64,7 +64,7 @@ def resolve_packages(request: Request) -> RequestOutput:
 def _resolve_packages(request: Request) -> RequestOutput:
     """Run all requested package managers, return their combined output."""
     _supported_package_managers = _package_managers
-    requested_types = set(pkg.type for pkg in request.packages)
+    requested_types = {pkg.type for pkg in request.packages}
```

Comment:
Non-blocking question: was this change suggested by the tool? (I have a preference for using `set` explicitly because of `{}`'s ability to create confusion. While the notation makes total sense historically it does bite unsuspecting users quite often.)